usecases/monitoring/prometheus.go

```diff
@@ -138,6 +138,12 @@ type PrometheusMetrics struct {
 	T2VRateLimitStats     *prometheus.GaugeVec
 	T2VRepeatStats        *prometheus.GaugeVec
 	T2VRequestsPerBatch   *prometheus.HistogramVec
+
+	TokenizerDuration           *prometheus.HistogramVec
```

Comment:
Curious can we add those metrics close to the tokenizer itself?

Similar to [grpc metrics.](https://github.com/weaviate/weaviate/blob/main/cluster/rpc/server.go#L55) (close to the actual server). Rationale being, keeping the metrics close to the actual components makes the component self contained (easy to move, mock and reason about and avoid coupling this single huge struct to all the components). 

Same reason [we marked this global Prometheus struct as deprecated and new metrics are added to the corresponding components.](https://github.com/weaviate/weaviate/blob/main/usecases/monitoring/prometheus.go#L36-L38)

Reply:
Let's do it later, we're going to have to rework that entire file anyway